helpme/helpme.py

```diff
@@ -896,7 +897,16 @@ def give_history_advice(plugin):
 Senior Code Monkey."""
 
 
+# Format hint `simple` makes lightning-cli print it as (-H) human readable
+def format_simple(fn):
```

Comment:
Nice one, we might want to add that to pylightning directly so it can be used by all plugins :+1: